dashboard/test/controllers/api/v1/ml_models_controller_test.rb

```diff
@@ -5,8 +5,6 @@ def stub_firehose
     FirehoseClient.instance.stubs(:put_record).with do |stream, args|
       @firehose_record = args
       @firehose_stream = stream
-      puts args
-      puts stream
```

Comment:
This is just noise in the test log. I ran into it while testing this. This module also tests its integration with the PII/Profanity detection.